modules/fluid_properties/test/tests/ics/rho_from_pressure_temperature/tests

```diff
@@ -0,0 +1,8 @@
+[Tests]
+  [./test]
```

Comment:
We should go ahead and remove dots in these files too

Reply:
ok